handler/src/main/java/io/netty/handler/ssl/OpenSslEngine.java

```diff
@@ -858,7 +860,7 @@ public synchronized SSLEngineResult unwrap(ByteBuffer src, ByteBuffer dst) throw
     }
 
     @Override
-    public synchronized SSLEngineResult unwrap(ByteBuffer src, ByteBuffer[] dsts) throws SSLException {
+    public final synchronized SSLEngineResult unwrap(ByteBuffer src, ByteBuffer[] dsts) throws SSLException {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Consider using varargs for methods or constructors which take an array the last parameter. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=pmd%3AUseVarargs)